pkgs/tools/admin/ossutil/use_go_mod.patch

```diff
@@ -0,0 +1,125 @@
+From 64067e979fb24ffb198a0c4eca718e81b63f514e Mon Sep 17 00:00:00 2001
```

Comment:
Is that an upstream patch we can fetch?

Reply:
So I got this patch from adding a commit to [my fork of that repo](https://github.com/jpetrucciani/ossutil/commit/64067e979fb24ffb198a0c4eca718e81b63f514e) (and originally I was fetching directly from that, but a friend suggested I just add the patch into my PR here) - I do plan on trying to make a PR to the original repo, however there was a [previous PR trying to make that change](https://github.com/aliyun/ossutil/pull/147/files) as well that hadn't been replied to that was no longer working (as the dependencies had changed on the main repo)

Reply:
Can you do a PR to the repo and fetch that instead? That is better for big patches to keep the repo size small.

Reply:
For sure!

Reply:
This patch file has been removed, I've opened a PR on the original repo [here](https://github.com/aliyun/ossutil/pull/155), and this now uses fetchpatch!